backend/configs/meta.json

```diff
@@ -3,8 +3,8 @@
   "dev_host": "http://localhost:3000",
   "name": "Aura 24",
   "emails": {
-    "primary": "aura23.klsgit@gmail.com",
```

Comment:
Pick this up from the env so that any changes we make due to the email limit is reflected here without the code being changed.

Reply:
![](https://github.com/sudeepk11/aura-git/assets/25717503/7c9a92b6-8670-45f4-8022-24899fcc0e1d)

The usages are only applicable for specifying an email address to reply to, in the emails. So in short these are not used by the nodemailer in any way, so no worries.